modules/statics/src/base.ts

```diff
@@ -542,9 +542,11 @@ export enum UnderlyingAsset {
   RDN = 'rdn',
   REB = 'reb',
   REBL = 'rebl',
+  REEF = 'reef',
   REP = 'rep',
   REPV2 = 'repv2',
   RFR = 'rfr',
+  RFUEL = 'refuel',
```

Comment:
Should this be `rfuel` instead?
```suggestion
  RFUEL = 'rfuel',
```